docs/src/operations/disk-space.md

```diff
@@ -0,0 +1,15 @@
+# Disk space
+
+Zeebe uses the local disk for storage of it's persistent data. Therefore if the Zeebe broker runs out of disk space the system is in an invalid state, as the broker cannot
+update it's state.
+
+To prevent the system to end in an unrecoverable state Zeebe expects a minimum size of free disk space available, and if this limit violated the broker will reject new
+requests, to allow the operations team to free more disk space to allow the broker to continue to update it's state.
+
+Zeebe can be configured with the following settings for the disk usage watermarks:
+
+* **zebee.broker.data.diskUsageReplicationWatermark**: the fraction of free disk space required for replication (default: 0.99)
```

Comment:
```suggestion
* **zeebe.broker.data.diskUsageReplicationWatermark**: the fraction of free disk space required for replication (default: 0.99)
```

Reply:
The value is the fraction of disk space that can be used/filled before we stop replication.